jujugui/static/gui/src/app/components/env-size-display/env-size-display.js

```diff
@@ -32,6 +32,19 @@ class EnvSizeDisplay extends React.Component {
   */
   _genClasses(section) {
     const guiState = this.props.appState.current.gui;
+    let isAppTab = false;
+
+    if (guiState) {
+      if (guiState[section] !== 'machine' && guiState[section] !== 'status') {
```

Comment:
shouldn't this be:

```JS
if (!guiState || guiState[section] === undefined) {
```

This would cover if the state doesn't exist, and if neither the status or machines exist in state. It would also do the correct thing in the application situation as it will always be undefined.

Reply:
No, it just makes all tabs active.